easybuild/easyconfigs/d/DMTCP/DMTCP-2.6.0-20210326.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'DMTCP'
+version = '2.6.0'
+local_commit = 'd0cec6b2ce32201e6b4de6a478d3f3f4032ac121'
+local_commit_date = '20210326'
+versionsuffix = '-%s' % (local_commit_date)
+
+homepage = 'http://dmtcp.sourceforge.net/index.html'
+description = """DMTCP is a tool to transparently checkpoint the state of multiple
+simultaneous applications, including multi-threaded and distributed applications.
+It operates directly on the user binary executable, without any Linux kernel modules
+or other kernel modifications."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/dmtcp/dmtcp/archive/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['b62f6fa612266401e27c70ab9f6515dd2588ad37c019ec73f691e5d5ea2b4c74']
+
+builddependencies = [('Autotools', '20220317')]
+
+sanity_check_paths = {
+    'files': ['bin/dmtcp_command', 'bin/dmtcp_discover_rm', 'bin/dmtcp_nocheckpoint', 'bin/dmtcp_srun_helper',
+              'bin/dmtcp_sshd',  'bin/dmtcp_coordinator', 'bin/dmtcp_launch', 'bin/dmtcp_restart', 'bin/dmtcp_ssh'],
+    'dirs': [],
+}
+
```

Comment:
Do we have any simple sanity check command to run
```
sanity_check_commands = ['dmtcp_command -h']
```
something like this

Reply:
Yep, should be able to add that.